src/app/components/Footer/index.tsx

```diff
@@ -54,27 +56,35 @@ export const Footer = memo(() => {
           defaults="<TermsLink>Terms and Conditions</TermsLink>"
         />
       </Text>
-      {process.env.REACT_APP_BUILD_DATETIME && process.env.REACT_APP_BUILD_SHA && (
-        <Text size="small" textAlign="center">
-          <Trans
-            i18nKey="footer.version"
-            t={t}
-            components={{
-              CommitLink: (
-                <Anchor
-                  href={`${githubLink}commit/${process.env.REACT_APP_BUILD_SHA}`}
-                  label={process.env.REACT_APP_BUILD_SHA.substring(0, 7)}
-                />
-              ),
-            }}
-            defaults="Version: <CommitLink/> built at {{buildTime}}"
-            values={{
-              buildTime: intlDateTimeFormat(Number(process.env.REACT_APP_BUILD_DATETIME)),
-            }}
-          />
-          {poweredByLabel && <Box align="center">{poweredByLabel}</Box>}
-        </Text>
-      )}
+      {process.env.REACT_APP_BUILD_DATETIME &&
+        process.env.REACT_APP_BUILD_SHA &&
+        process.env.REACT_APP_BUILD_VERSION && (
+          <Text size="small" textAlign="center">
+            <Trans
+              i18nKey="footer.version"
+              t={t}
+              components={{
+                ReleaseLink: (
+                  <Anchor
+                    href={githubReleaseLink(process.env.REACT_APP_BUILD_VERSION)}
```

Comment:
Shouldn't we open external links in a new tab with `rel="noopener"` etc?

Reply:
Another issue not related to this PR, but are these links actually work in extension?

Reply:
One case to consider is to try out `noreferrer` in the wallet selected case. Where if the existing links are sending the referer, we might be actually exposing user's public key.